ppci/cli/link.py

```diff
@@ -56,13 +56,16 @@ def link(args=None):
     """ Run linker from command line """
     args = parser.parse_args(args)
     relocatable = args.relocatable
+    print("lib=%s" % args.library)
     with LogSetup(args):
+        # TSF linker library issue: add libraries arg
```

Comment:
Should this comment remain?

Reply:
No, I was not sure if you have got a convention for fix comments (TSF are my initials)

Reply:
there is no real convention for this, if you find this handy, then it remains. I've done this before in other places where I placed a note. Almar did something similar.